packages/utils/src/address.ts

```diff
@@ -12,4 +40,9 @@ export const privateKeyToAddress = (privateKey: string) => {
   )
 }
 
+export const privateKeyToPublicKey = (privateKey: string) => {
+  return toChecksumAddress(
```

Comment:
Did you mean to call toChecksumAddress here?

Reply:
Yeah, any reason not to?

Reply:
I thought checksums only apply to addresses, not public keys